src/patterns/instance-role.ts

```diff
@@ -10,7 +10,8 @@ import {
   GuSSMRunCommandPolicy,
 } from "../constructs/iam";
 
-interface InstanceRoleProps extends GuGetS3ObjectPolicyProps, Partial<GuLogShippingPolicyProps> {
+interface InstanceRoleProps extends GuGetS3ObjectPolicyProps {
+  includeLogShipping: boolean;
```

Comment:
Would it be worth providing a default value for this?

Reply:
Yes! Not sure how to achieve this in a nice way though. Maybe relying on javascript's ~insane~ logic of `undefined` being falsy and inverting the prop name?

```typescript
interface InstanceRoleProps {
  withoutLogShipping? boolean
}
```

Then the ternary becomes `...(props.withoutLogShipping ? [] : [new GuLogShippingPolicy(scope)])`.

That is, make it opt out.

Reply:
Hmm... yeah - I was thinking of spreading the props with a default `{withLogShipping: true, ...props}` although I'm not sure where you'd do that. Maybe it'd need to be a new `const` before the `this.policies` line which isn't great. 

Having it as `withoutLogShipping` makes it more obvious to reason about what the default behaviour is though, I like it!

Reply:
Updated now.